cmd/kaspawallet/config.go

```diff
@@ -63,6 +64,11 @@ type broadcastConfig struct {
 	config.NetworkFlags
 }
 
+type showAddressConfig struct {
+	KeysFile string `long:"keys-file" short:"f" description:"Keys file location (only if different from default)"`
```

Comment:
Instead of "(only if different from default)" better to write something like "(default: ~/.kaspawallet/blabla.json)".